packages/frontend/src/Views/Home/Pagination/reducer/actions/pagination.actions.ts

```diff
@@ -0,0 +1,24 @@
+import { Dispatch } from "react";
+import { PaginationActions } from "../../paginationActions.enum";
+import {
+  SetTagsArrAction,
+  SetFilterTextAction,
+  SetMaxPagesAction,
+  SetPageAction,
+} from "../types/paginationActions.types";
+
+export const setPage = (dispatch: Dispatch<SetPageAction>, page: number) => {
```

Comment:
Moim zdaniem to jest złe rozwiązanie. Tak nie powinno się wywoływać akcji na reducerze.   
Po co przekazywać tego dispatcha w argumentach jak można zrobić w komponencie:  
`dispatch(SetPage(page))`, a tak musisz przekazywać `setPage(dispatch, page)`.   
Pierwszy zapis jest dla mnie o wiele czytelniejszy.

Reply:
po to że możesz dopisać jakąś logike w fukncji zapis nie jest błędny a wręcz bardzie poprawny niż określenie akcji w środku, mając dispatch w śrlodku tej funkcji możesz wywołać więcej niż jedną akcje, np dojdzie nam loading state do paginacji to nie musisz tworzyć kolejnej akcji tylko używasz dispatch, cała logike związaną ze stanem zamykasz w akcji a nie wyciągasz na widok